### DISCUSSION TITLE
'Framing' needed for Calc groups added to DirectLake models

### DISCUSSION THREAD

**Discussion by m-kovalsky** [TabularEditor/TabularEditor3#1197] [Ideas]
**Describe the bug**
After creating/modifying a Calculation Group in a DirectLake model, a 'framing' operation is needed. TE3 should give a warning that the Calc Group table is unprocessed and needs to be processed in order to work.

**To Reproduce**
1. Open a DirectLake model connected to the PBI service
2. Create a calc group & calc item(s)
3. Save the model back to the service
4. No warning shown regarding the need to process the calc group

**Application specifics**
 - Tabular Editor 3.11.0

**Comment by otykier** [comment]
Hi Michael,
We are treating this as a feature request rather than a bug, to be included in an update in the near future. More generally, we will display warning messages when any object in the model is in an unprocessed state.

**Comment by m-kovalsky** [comment]
That's great Daniel. I think it will be a nice feature to add to TE3.

**Comment by mlonsk** [comment]
The feature to show all objects that are in a unprocessed state has been added in Tabular Editor 13.3.0
Closing the feature request discussion 😁🔏

**Comment by m-kovalsky** [comment]
Thanks for adding this. I believe it will be quite useful for users. One more suggestion would be to allow right clicking on one of these notifications in the messages window and showing the option to solve the issue by processing the object(s) accordingly.
